src/vellum/client/types/local_type_definition.py

```diff
@@ -0,0 +1,9 @@
+from typing import Any, Dict
```

Comment:
we should **never** edit files in the `src/vellum/client` directory directly. Move these types to some file from within the workflows module for now